modules/search/instant-search/components/search-sort-widget.jsx

```diff
@@ -0,0 +1,49 @@
+/** @jsx h */
+
+/**
+ * External dependencies
+ */
+import { h, Component } from 'preact';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { getSortOptions } from '../lib/sort';
+
+export default class SearchSortWidget extends Component {
+	constructor( props ) {
+		super( props );
+		this.state = { selected: this.props.initialValue };
+		this.handleChange = this.handleChange.bind( this );
+	}
+
+	handleChange( event ) {
+		this.setState( { selected: event.target.value }, () => {
+			this.props.onChange( event.target.value );
+		} );
+	}
+
+	render() {
+		const sortOptions = getSortOptions();
+		return (
+			<label>
+				{ __( 'Sort by', 'jetpack' ) }
+				<select
+					className="jetpack-instant-search__sort-widget-select"
+					onBlur={ this.handleChange }
```

Comment:
Wouldn't this fire off redundant calls to `handleChange` even when the selected value hasn't changed?

Reply:
Indeed it will. It's to satisfy this lint requirement, which is up for discussion:

https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/no-onchange.md